### PR TITLE
Don't install images if they already exist.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: tools/run-image.toit [hunk 1]
```diff
@@ -57,16 +68,20 @@ main arguments:
     print_ "Usage: run-image <image|directory>"
     exit 1
 
+  existing-uuids/Set := {}
```
Comment (kasperl) [inline tools/run-image.toit:71]: Maybe this part of the setup should be in a separate method, so we don't keep the new set alive while we're calling `boot`?

File: tools/run-image.toit [hunk 2]
```diff
@@ -41,7 +43,16 @@ class SystemImage extends ContainerImage:
   delete -> none:
     unreachable  // Not implemented yet.
 
-add-image path/string -> none:
+add-image path/string existing-uuids/Set -> none:
```
Comment (kasperl) [inline tools/run-image.toit:46]: I'd probably move this below main.

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: tools/run-image.toit (hunk 2 above)
Comment (floitsch) [reply tools/run-image.toit:59]: done.

File: tools/run-image.toit (hunk 1 above)
Comment (floitsch) [reply tools/run-image.toit:75]: done.

**Pull request merged**